### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lalitmee**
Adding one more resource for Fundamentals of Algorithms

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Community (college) maintained list of algorithm implementations and codes.
 * [Algorithms - Learneroo](https://www.learneroo.com/subjects/8)
 * [Awesome-Algorithms](https://github.com/tayllan/awesome-algorithms)
 * [Stanford-ACM-Codes](https://github.com/jaehyunp/stanfordacm) - A list of codes written by previous Stanford ACM team members and coaches.
+* [Fundamentals of Algorithms - GeekforGeeks](http://www.geeksforgeeks.org/fundamentals-of-algorithms/)
```
Comment (yashLadha) [inline README.md]: It should be in alphabetical order. Should come before Stanford-ACM-Codes.

Comment (yashLadha) [inline README.md]: It should be in alphabetical order. Should come before Stanford-ACM-Codes?

Comment (aviaryan) [inline README.md]: The title is a bit misleading. Make it something like `Algorithm list - GeeksForGeeks` .

Comment (aviaryan) [inline README.md]: I don't think this is strictly required but yes, it will help.

Comment (lalitmee) [inline README.md]: Then, all resources should be in alphabetical order.

Comment (lalitmee) [inline README.md]: Okk.

File: README.md [hunk 2]
```diff
@@ -15,12 +15,13 @@ Community (college) maintained list of algorithm implementations and codes.
 
 ## Resources
 
-* [Popular Data Structures and Algorithms - Codechef](https://discuss.codechef.com/questions/48877/data-structures-and-algorithms)
 * [Intro to Algorithms - Khan Academy](https://www.khanacademy.org/computing/computer-science/algorithms)
```
Comment (yashLadha) [inline README.md]: Intro to Algorithms - Khan Academy should come after Fundamentals of Algorithms - GeekforGeeks

**Comment by yashLadha** [comment]
Sir, It would be good if you squash your commits. So that we have one or two commits per PR.

File: README.md (hunk 1 above)
Comment (aviaryan) [inline README.md]: @lalitmee You haven't changed the link title yet.

Comment (lalitmee) [inline README.md]: Done.

File: README.md [hunk 3]
```diff
@@ -15,13 +15,15 @@ Community (college) maintained list of algorithm implementations and codes.
 
 ## Resources
 
-* [Popular Data Structures and Algorithms - Codechef](https://discuss.codechef.com/questions/48877/data-structures-and-algorithms)
-* [Intro to Algorithms - Khan Academy](https://www.khanacademy.org/computing/computer-science/algorithms)
 * [Algorithms - Learneroo](https://www.learneroo.com/subjects/8)
 * [Awesome-Algorithms](https://github.com/tayllan/awesome-algorithms)
+* [Algorithms List - GeekforGeeks](http://www.geeksforgeeks.org/fundamentals-of-algorithms/)
```
Comment (aviaryan) [inline README.md]: It should be `GeeksForGeeks` . 
Also please change the PR title to reflect the change done in this PR.

**Comment by yashLadha** [comment]
@aviaryan there should be one commit per PR? What do u think?

File: README.md (hunk 3 above)
Comment (lalitmee) [inline README.md]: But the website has name `GeekforGeeks`.

Comment (singhpratyush) [inline README.md]: screenshot from 2017-03-09 22-45-38
 Not quite so..

**Comment by aviaryan** [comment]
@yashLadha That was needed earlier because maintainers didn't had the time to checkout and squash each PR. But now we have the squash feature on the GitHub website. So that's not really a problem now.

**Pull request merged**